### ISSUE TITLE
A11y

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-azurefunctions#3396]
**GitHubTags:**

#A11ySev2; #TCS-E+d; #NFT-Fluentui-Web-Oct2022; #DesktopWeb; #WCAG1.4.10.
#Reflow; #A11yMAS.

**OS Details:**  

Web: win11

**Environment Details:** 

Application Name: Fluent Ui

**Repro Steps:** 

1.Hit enter on the URL https://react-dev.fluentui.dev/
2."Fluent UI React v9" page will open.
3.Tab till "Components" button and hit enter.
4.Tab till "Avatar" link on the left side bar and hit enter.
5.verify after applying reflow avatar page is not visible clearly in whole page 

**Actual Result:** 

when we apply reflow avatar page is not visible clearly in whole page
Similar issue is observed in whole application.

**Expected Result:** 

when we apply reflow avatar page should be visible clearly in whole page

**User Impact:** 

Reflow users wil get difficulties when avatar page is not visible clearly in whole page

![3396A11y_Fluent UI_Components_Avatar_Reflow](https://user-images.githubusercontent.com/89508906/198254709-43cd8701-f0ba-4b8b-a798-7cb8852fb285.gif)

**Comment by Latha2564** [comment]
Bug is closed as bug logged in wrongADO
